app/questionnaire_state/state_repeating_answer_question.py

```diff
@@ -44,6 +47,25 @@ def create_new_answer_state(self, answer_schema, answer_instance, group_instance
     def add_new_answer_state(self, answer_state):
         self.answers.append(answer_state)
 
+    def answers_grouped_by_instance(self):
```

Comment:
I wonder if this should be a top level function somewhere as every answer has an answer_instance it's just repeating answers which increment that version